llvm/lib/ObjectYAML/MinidumpEmitter.cpp

```diff
@@ -136,6 +136,22 @@ static size_t layout(BlobAllocator &File, MinidumpYAML::ExceptionStream &S) {
   return DataEnd;
 }
 
+static size_t layout(BlobAllocator &File, MinidumpYAML::Memory64ListStream &S) {
+  size_t BaseRVA = File.tell() + sizeof(minidump::Memory64ListHeader);
```

Comment:
I agree with this, but because this entire class is on `size_t` is it a good idea to refactor this one method to return a `uint64_t`? Below in `layout(BlobAllocator &File, Stream &S)` everything either returns an `size_t` or void. 

If we refactor, I'd convert the local variable `DataEnd` in that layout function to use `uint64_t` as well.